eureka-client/build.gradle

```diff
@@ -18,6 +18,8 @@ dependencies {
     compile "org.apache.httpcomponents:httpclient:${apacheHttpClientVersion}"
     compile "com.google.inject:guice:${guiceVersion}"
 
+    compile "com.github.vlsi.compactmap:compactmap:1.2.1"
```

Comment:
It looks like this library is licensed with LGPL. I think there was general advice to keep everything Apache 2 if possible so the set of licenses consumers need to worry about is minimal.